generator/generator.go

```diff
@@ -29,105 +30,112 @@ func createDir(storagePath string, filePath string) error {
 		if err != nil {
 			return fmt.Errorf("unable to create path: %s", storagePath+path)
 		}
+		return nil
```

Comment:
I believe this line is unnecessary.

Reply:
If we remove this line we get ``missing return at end of function`` error.